code/datums/uplink_item.dm

```diff
@@ -410,6 +410,13 @@ var/list/uplink_items = list()
 	item = /obj/item/weapon/twohanded/chainsaw
 	cost = 13
 
+/datum/uplink_item/dangerous/batterer
+	name = "Mind Batterer"
+	desc = "A device that has a chance of knocking down people around you for a long amount of time.50% chance per person. The user is unaffected. Has only two charges."
```

Comment:
It's a bit silly to leave this as a description, a description is supposed to be what you see of the object, not a tutorial.
Last time I used a spoon it didn't tell me how to use it when I looked at it

You can have this information in the uplink instead